godu.go

```diff
@@ -29,13 +31,50 @@ func main() {
 	s := initScreen()
 	commands := make(chan core.Executer)
 	states := make(chan core.State)
+	lastStateChan := make(chan *core.State, 1)
 	var wg sync.WaitGroup
 	wg.Add(3)
-	go core.StartProcessing(tree, commands, states, &wg)
+	go core.StartProcessing(tree, commands, states, lastStateChan, &wg)
 	go InteractiveTree(s, states, &wg)
 	go ParseCommand(s, commands, &wg)
 	wg.Wait()
 	s.Fini()
+	lastState := <-lastStateChan
+	printMarked(lastState)
+}
+
+func printMarked(state *core.State) {
```

Comment:
could you please move this to a separate file in `core` and write a test for it? It should be easily testable if you change the signature to `printMarked(state *core.State, writer io.Writer)` Example of such a test is in older `godu` version (https://github.com/viktomas/godu/blob/a7bb16532ebe4e303ed6f4b0845774c7b90b64f2/interactive/interactive_test.go)